test/e2e/horizontal_pod_autoscaling.go

```diff
@@ -26,37 +27,57 @@ import (
 )
 
 const (
-	kind             = "replicationController"
+	kindRC           = "replicationController"
+	kindDeployment   = "deployment"
 	subresource      = "scale"
 	stabilityTimeout = 10 * time.Minute
 )
 
 var _ = Describe("Horizontal pod autoscaling", func() {
+
 	var rc *ResourceConsumer
 	f := NewFramework("horizontal-pod-autoscaling")
 
-	// CPU tests
-	It("[Skipped][Autoscaling Suite] should scale from 1 pod to 3 pods and from 3 to 5 (scale resource: CPU)", func() {
-		rc = NewDynamicResourceConsumer("rc", 1, 250, 0, 500, 100, f)
-		defer rc.CleanUp()
-		createCPUHorizontalPodAutoscaler(rc, 20)
-		rc.WaitForReplicas(3)
-		rc.EnsureDesiredReplicas(3, stabilityTimeout)
-		rc.ConsumeCPU(700)
-		rc.WaitForReplicas(5)
+	titleUp := "[Skipped][Autoscaling Suite] should scale from 1 pod to 3 pods and from 3 to 5 (via %s, with scale resource: CPU)"
```

Comment:
Also could you please disable deployment tests by default by removing [Autoscaling Suite] tag?